src/android.rs

```diff
@@ -24,9 +24,11 @@ pub fn open_browser_default(url: &str, options: &BrowserOptions) -> Result<()> {
     }
 
     // Create a VM for executing Java calls
-    let native_activity = ndk_glue::native_activity();
-    let vm_ptr = native_activity.vm();
-    let vm = unsafe { jni::JavaVM::from_raw(vm_ptr) }.unwrap();
+    let ctx = ndk_context::android_context();
+    let vm = unsafe {
+        jni::JavaVM::from_raw(ctx.vm() as _).expect("Expected to find JVM via ndk_context crate")
```

Comment:
Or is it better to return something like `Error::new(ErrorKind::Other, "Failed to find JVM via ndk_context crate")` ?

Reply:
oh, just to clarify, the previous `unwrap()` wasn't the cause of any crash for me.

I was getting a panic simply due to the fact that `ndk-glue` was being linked into my application and the panic would have been a conflict between `ndk-glue` and `android-activity`. (I.e. the purpose of this change isn't about avoiding a panic with this specific code it's just about removing the dependency on the `ndk-glue` crate).

If nothing has initialized `ndk-context` I think that is significant enough of an issue that it's hard to imagine that the app is in a consistent state at all and panicking, as it would have done previously, seems perfectly reasonable to me.